pre-run-files.yaml

```diff
@@ -0,0 +1,6 @@
+- hosts: localhost
```

Comment:
This could be called generate-clouds.yaml to match the generate-inventory playbook.